<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt做的一个程序</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>530</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>512</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string>等待nii文件加载</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>推理</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>保存原图</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>保存推理后的图像</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
